packages/fluentui/docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -31,14 +31,19 @@ const behaviorMenu: ComponentMenuItem[] = require('../../behaviorMenu');
 
 const componentsBlackList = ['Debug', 'Design'];
 
-class Sidebar extends React.Component<any, any> {
+interface SidebarState {
+  query: string;
+  activeCategoryIndex: number | number[];
+}
+
+class Sidebar extends React.Component<any, SidebarState> {
```

Comment:
It would be nice to add typings for `props` as well.

Reply:
Added typings for `props`